addons/website/static/src/js/content/inject_dom.js

```diff
@@ -35,6 +35,9 @@ document.addEventListener('DOMContentLoaded', () => {
     for (const conditionalEl of conditionalEls) {
         const selectors = conditionalEl.dataset.visibilitySelectors;
         styleEl.sheet.insertRule(`${selectors} { display: none !important; }`);
+        // Remove the data attribute responsible of the crossed-out eye icon
+        // displayed in edit mode.
+        delete conditionalEl.dataset.invisible;
```

Comment:
For me this shouldn't be done here, the `data-invisible` is useless outside of edit mode right ?
If yes, this attribute should be removed when we leave the edit mode

Reply:
I moved this part in `cleanForSave`